src/roiextractors/extractors/hdf5imagingextractor/hdf5imagingextractor.py

```diff
@@ -36,6 +37,8 @@ def __init__(
         channel_names: ArrayType = None,
     ):
         ImagingExtractor.__init__(self)
+        self._images_in_standard_structure = True
```

Comment:
The images will return the proper structure, so ordering items are strictly for internal/private usage, so IDK that we need to track it

Reply:
We wouldn't want anything downstream to use a class flag, even a private one, that is only expected to be temporary.

Reply:
Eliminated the flags.